OneMore/Commands/File/Markdown/MarkdownConverter.cs

```diff
@@ -51,6 +52,28 @@ public void RewriteHeadings()
 			{
 				RewriteHeadings(outline.Descendants(ns + "OE"));
 			}
+
+			// added header spacing specific to markdown
+			var quickstyles = page.Root.Elements(ns + "QuickStyleDef");
```

Comment:
This is a bad assumption. Should only affect Outline where we are converting markdown. Should abide by existing quick style definitions for page. If user wants spacing, they can create their own custom OneMore Style set and apply that after the conversion.